lib/src/native/realm_core.dart

```diff
@@ -581,8 +581,8 @@ class _RealmCore {
     });
   }
 
-  void realmSetAutoRefresh(Realm realm, bool enable) {
-    _realmLib.realm_set_auto_refresh(realm.handle._pointer, enable);
+  void realmDisableAutoRefreshForTesting(Realm realm) {
```

Comment:
I changed this method name in `session` class and removed the parameter. I think that the parameter `enabled` should be available here in realm_core.dart. We just expose the C-API in this class, we don't have to support smart logic on this level in my oppinion.

Reply:
please, see the next comment